src/domain/interfaces/transaction-api.interface.ts

```diff
@@ -151,4 +151,11 @@ export interface ITransactionApi {
   ): Promise<Estimation>;
 
   getMessageByHash(messageHash: string): Promise<Message>;
+
+  getMessagesBySafe(
+    safeAddress: string,
+    ordering?: string,
```

Comment:
I don't think we use the `ordering` parameter at all for this feature. I think it's better to remove it to keep the number of parameters smaller.